nixos/modules/programs/wayland/sway.nix

```diff
@@ -42,6 +42,11 @@ in {
       <https://github.com/swaywm/sway/wiki> and
       "man 5 sway" for more information'');
 
+    enableRealtime = mkEnableOption (lib.mdDoc ''
+      add CAP_SYS_NICE capability on `sway` binary for realtime scheduling
+      privileges. This may improve latency and reduce stuttering, specially in
+      high load scenarios'') // { default = true; };
```

Comment:
Defaulting to `true` after the patch because as far I understand there shouldn't be any more issues here.

WDYT? @Emantor.

Reply:
Yep, I had this patch running for a few months before I switched to the limits `rtprio` approach.